### PR TITLE
Remove instructions for scoop

### PR DESCRIPTION
**Pull request by ivanayov**
Those instructions are outdated and are confusing users and
causing errors. Pointing to the latest exe file instead

Signed-off-by: Ivana Yovcheva (VMware) <iyovcheva@vmware.com>

## Motivation and Context

- [x] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))

Raised on slack

## How Has This Been Tested?

N/A 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,13 +66,7 @@ $version = (Invoke-WebRequest "https://api.github.com/repos/openfaas/faas-cli/re
...
 #### Windows
 
-The easiest way to install the faas-cli is through `scoop`:
-
-```
-scoop install faas-cli
-```
-
-Note: The `scoop` release may not run the latest minor release but is updated regularly.
+To install the faas-cli on Windows go to Releases and download the latest faas-cli.exe.
```
Comment (alexellis) [inline README.md]: Can you make this a link please to the Releases page?

**Comment by alexellis** [comment]
LGTM, thank you.

**Comment by itscaro** [comment]
Hello @ivanayov , what are the errors caused?

**Comment by ivanayov** [comment]
Hi @itscaro 
The context is now a bit lost in the Slack history, but someone had issues with this documentation and as it's not regularly updated, we decided to remove it.

**Comment by alexellis** [comment]
This was reported on Twitter - the version was very much out of date and contained a bug which gave the user a poor experience of OpenFaaS. Since this isn't maintained by an active community member we opted to remove it.

Alex

**Comment by alexellis** [comment]
It does look like one of the maintainers of scoop or the scoop community took over from where you left off in January 2018:

https://github.com/lukesampson/scoop/commits/master/bucket/faas-cli.json

https://github.com/openfaas/faas-cli/releases

I'm unsure why the user had such an old version, but he said he'd installed it with scoop. I'm not aware of wide-spread use of scoop with the OpenFaaS CLI. Can we get any metrics? 

To sum up I'm not against the package being listed as long as it's maintained.

Alex
